recipe/meta.yaml

```diff
@@ -17,15 +17,16 @@ build:
 
 requirements:
   build:
-    - toolchain
+    - {{ compiler('c') }}
+  host:
     - python
     - setuptools >=18.0
     - cython >=0.25.2
-    - numpy >=1.11.3,<1.12
+    - numpy
```

Comment:
It's trying to install [too old of a version of `numpy`]( https://travis-ci.org/conda-forge/cyminmax-feedstock/jobs/395693790#L343 ), which is problem. Basically this version constraint is necessary for it to build.